docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,328 @@
+---
+
+layout: docs
+
+title:  "Connectable Operators"
+
+section: "chisel3"
+
+---
+
+The `Connectable` operators are the standard way to connect Chisel hardware components to one another.
```

Comment:
```suggestion
The `Connectable` operators are the standard way to connect Chisel hardware components to one another.
```

moving this question higher, what is a "component"? do people understand that term? Should we define it as "A chisel Data that is bound to hardware (IO, Reg, Wire, (dataview? lit?))